src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php

```diff
@@ -411,4 +411,259 @@ public function testSendPhotoWithOptions()
         $this->assertEquals(1, $sentMessage->getMessageId());
         $this->assertEquals('telegram://api.telegram.org?channel=testChannel', $sentMessage->getTransport());
     }
+
+    public function testSendPhotoByFileIdWithOptions()
+    {
+        $response = $this->createMock(ResponseInterface::class);
+        $response->expects($this->exactly(2))
+            ->method('getStatusCode')
+            ->willReturn(200);
+
+        $content = <<<JSON
+            {
+                "ok": true,
+                "result": {
+                    "message_id": 1,
+                    "from": {
+                        "id": 12345678,
+                        "is_bot": true,
+                        "first_name": "YourBot",
+                        "username": "YourBot"
+                    },
+                    "chat": {
+                        "id": 1234567890,
+                        "first_name": "John",
+                        "last_name": "Doe",
+                        "username": "JohnDoe",
+                        "type": "private"
+                    },
+                    "date": 1459958199,
+                    "photo": [
+                        {
+                            "file_id": "ABCDEF",
+                            "file_unique_id" : "ABCDEF1",
+                            "file_size": 1378,
+                            "width": 90,
+                            "height": 51
+                        },
+                        {
+                            "file_id": "ABCDEF",
+                            "file_unique_id" : "ABCDEF2",
+                            "file_size": 19987,
+                            "width": 320,
+                            "height": 180
+                        }
```

Comment:
Why adding 2 photos?

Reply:
This is wrong, will be fixed.
Copy pasted from previous test, will fix it too.